test/e2e_node/log_path_test.go

```diff
@@ -39,6 +39,25 @@ var _ = framework.KubeDescribe("ContainerLogPath", func() {
 	f := framework.NewDefaultFramework("kubelet-container-log-path")
 	Describe("Pod with a container", func() {
 		Context("printed log to stdout", func() {
+			BeforeEach(func() {
+				if framework.TestContext.ContainerRuntime == "docker" {
+					// Container Log Path support requires JSON logging driver.
+					// It does not work when Docker daemon is logging to journald.
+					d, err := getDockerLoggingDriver()
+					framework.ExpectNoError(err)
+					if d != "json-file" {
+						framework.Skipf("Skipping because Docker daemon is using a logging driver other than \"json-file\": %s", d)
+					}
+					// Even if JSON logging is in use, this test fails if SELinux support
```

Comment:
If there any option we can set on the pod to prevent this failure? Such as `privileged`, or some selinux options, such as: 
```golang
PodSecurityContext: &v1.PodSecurityContext{
        SELinuxOptions: &v1.SELinuxOptions{                                                                                                                                              
                Level: "s0",
        },
},
```